ffv1.md

```diff
@@ -375,7 +375,7 @@ SVGI:![svg](samplediff.svg "coding of the sample difference")
 SVGI:!---
 SVGC:samplediff.svg=$$coder\\\_input=[(sample\\\_difference+2^{bits-1})\\&(2^{bits}-1)]-2^{bits-1}$$
 AART:coder_input =
-AART:    [(sample_difference + 2 ^ (bits-1)) & (2 ^ bits - 1)] - 2 ^ (bits - 1)
+AART:[(sample_difference + 2^(bits-1)) & (2^bits - 1)] - 2^(bits - 1)
```

Comment:
I suggest to unify the two `(bits-1)` without spaces around the minus sign. In my opinion, it makes the last one much more readable, especially considering the different one in the middle.